client/state/selectors/get-store-referrers-by-referrer.js

```diff
@@ -6,9 +6,10 @@ import { find } from 'lodash';
 /**
```

Comment:
Note: I didn't move this file under `state/stats` since it's only used in woocommerce code.